submissions/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class SubmissionsConfig(AppConfig):
```

Comment:
Sorry to be nitpicky, but I would keep with the naming convention and call prefix your `Submission` classes with `HackProject` to make it `HackProjectSubmissionConfig` and `HackProjectSubmissionForm`.